Documentation/user-guides/getting-started.md

```diff
@@ -330,9 +162,5 @@ spec:
   resources:
     requests:
       memory: 400Mi
-  enableAdminAPI: true
+  enableAdminAPI: false
```

Comment:
Same thing here, not sure if resources and AdminAPI are relevant for this page's example